.circleci/config.yml

```diff
@@ -36,6 +36,15 @@ jobs:
       - run: sudo apt-add-repository -y ppa:rael-gc/rvm
       - run: sudo apt-get update && sudo DEBIAN_FRONTEND=noninteractive apt-get -y install rvm
 
+      - restore_cache:
+          keys:
+            - rust
+
+      - run: curl https://sh.rustup.rs -sSf | sh -s -- -y && cat ~/.cargo/env >> $BASH_ENV && source ~/.cargo/env && cargo --version && rustc --version
```

Comment:
please add a comment where you took this command (https://rustup.rs/) to avoid extra check of unknown shell script : )